scripts/shared/lib/shflags

```diff
@@ -4,13 +4,14 @@
 
 ensure_shflags() {
     if [ -f /usr/share/shflags/shflags ]; then
-	. /usr/share/shflags/shflags
-	return
+      . /usr/share/shflags/shflags
+      return
     fi
 
-    if [ ! -f .shflags ]; then
+    if [ ! -s .shflags ]; then
         echo "Downloading shflags ${SHFLAGS_VERSION}"
         if ! curl -L "https://raw.githubusercontent.com/kward/shflags/${SHFLAGS_VERSION}/src/shflags" > .shflags; then
+            rm -rf .shflags
```

Comment:
```suggestion
            rm -f .shflags
```

Reply:
I debated with myself about using the GH way to accept your patch and squash it locally. Turns out the answer was "yes, do that!". :stuck_out_tongue_winking_eye: